index.html

```diff
@@ -0,0 +1,174 @@
+                        consistency using ATP/WTA pro-level point patterns proven by data
+                    </div>
+                </div>
+
+                <div class="main-program-details-box3">
+                    <div class="main-program-details-img"> <img src="./img/main-prog3.png">
+                    </div>
+
+                    <div class="main-program-details-heading"> Forum </div>
+                    <div class="main-program-details-description"> For a limited time, you can get FREE access to this
+                        exclusive content right here.
+                    </div>
+                </div>
+                <div class="main-program-details-box4">
+                    <div class="main-program-details-img"> <img src="./img/main-prog4.png">
+                    </div>
+
+                    <div class="main-program-details-heading"> Workshop </div>
+                    <div class="main-program-details-description"> These sessions will show you the latest proven
+                        technical, tactical, mental and fitness secrets that the top coaches teach to their players.
+                    </div>
+                </div>
+
+                <div class="main-program-details-box5">
+                    <div class="main-program-details-img"> <img src="./img/main-prog5.png">
+                    </div>
+
+                    <div class="main-program-details-heading"> Tennis Ignite </div>
+                    <div class="main-program-details-description"> Tens of thousands of tennis players like you that
+                        have learned from summits. If you are ready to put in the work, we're here to show you the way!
+                    </div>
+                </div>
+
+                <button class="main-program-button">Join Tennis Summit 2023</button>
+            </div>
+
+        </div>
+        <h3> SEE THE WHOLE PROGRAM</h3>
+    </section>
+    <div class="featuredSpeakersHeading">Featured Speakers</div>
+    <div class="featuredSpeakersBar1"></div>
+    <section class="featuredSpeakers">
+
+    </section>
+
+    <section class="partners">
+        <div class="partners-heading">
+            <h1>Partner</h1>
+        </div>
+        <div class="partnersBar"></div>
+        <div class="partnersLogo">
+            <div class="mozilla"> <img src="./img/mozilla.png" alt="Dunlop Sport Logo"></div>
+            <div class="google"><img src="./img/Google-logo.png" alt="Dunlop Sport Logo"></div>
+            <div class="airbnb"><img src="./img/airbnb.png" alt="Dunlop Sport Logo"></div>
+            <div class="naver"><img src="./img/Naver.png" alt="Dunlop Sport Logo"></div>
+            <div class="twitter"><img src="./img/twitter.png" alt="Dunlop Sport Logo"></div>
+
+        </div>
+    </section>
+    </div>
+    <footer class="footer">
+        <div class="footerLogo">
+            <h1>&reg; Tennis Summit India</h1>
```

Comment:
- [ ] Good work writing semantic HTML, however, this is not the main heading of the page and can not, therefore, have `h1` tags. It is semantically wrong. Please change this to h2/h3 depending on your heading levels in the document. 🧑‍🏭

Reply:
Hello Francis,

Done.  Thanks